backend/sirius-web-spring-collaborative-forms/src/main/java/org/eclipse/sirius/web/spring/collaborative/forms/api/FormCreationParameters.java

```diff
@@ -33,8 +33,6 @@
 
     private UUID id;
 
-    private String label;
```

Comment:
Why are you removing the label?

Reply:
Because it has no use except to create a `FormCreationParameters` from another `FormCreationParameters`.